python/tools/duplicate_structure/README.md

```diff
@@ -0,0 +1,13 @@
+# Duplicate Structure
+
+A command line tool that duplicates a structure of objects in ftrack.
+
+## Usage
+
+```cmd
+python duplicate_structure.py <source_entity_id> <destination_entity_id>
```

Comment:
**destination_entity_id** is the parent where the **source_entity_id** will be created under ?

Reply:
Exactly.